test/e2e/specs/customer_admin.go

```diff
@@ -16,7 +16,7 @@ import (
 	"github.com/openshift/openshift-azure/test/clients/openshift"
 )
 
-var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin]", func() {
+var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fake]", func() {
```

Comment:
not needed

Reply:
It's actually needed now that [Fake] is used in the Makefile. Also these tests can only run against the Fake RP at the moment.